src/utils/puns.ts

```diff
@@ -19,7 +19,24 @@ const CLEAN_RUN_PUNS = [
   'Tank you very much!',
   'Why pun? Just for for the halibut.',
   "You've proven any-fin is possible.",
-  'You swim with the sharks!'
+  'You swim with the sharks!',
+  "You've found your sea legs.",
+  'You really made a splash!',
+  "You've earned a seal of approval.",
+  "You're swimming in success!",
+  "You're making waves now!",
+  "You've turned the tide.",
+  "You're catching on now!",
+  "You're in the swim of things now!",
+  "Catch of the day, right here!",
+  "That was a big fish to fry!",
+  "Nice job! Are you hooked yet?",
+  "You've found your sea legs!",
```

Comment:
This is the same as line 23 above (except `!` instead of `.`).

Reply:
cut line 34